src/firefly/js/core/background/BackgroundMonitor.jsx

```diff
@@ -58,8 +60,8 @@ export function showBackgroundMonitor(show=true) {
 
     return (
         <Stack justifyContent='space-between' spacing={1} alignItems='left' minHeight={200}
-               minWidth={330} maxWidth={590} sx={{resize: 'both', overflow: 'auto'}}>
-            <Box sx={{ flexGrow: 1, overflow: 'auto' }}>
+                    minWidth={330} maxWidth={590} sx={{resize: 'both', overflow: 'auto'}}>
```

Comment:
@robyww I made the other changes. I left this sx here with this Stack because merging it as <Stack {...{ _properties here_ }}> makes the reisze not work for the Background monitor popup (I am not sure why though).